src/admin/includes/modules/system/rth_stripe.php

```diff
@@ -15,14 +15,15 @@
  */
 
 use RobinTheHood\ModifiedStdModule\Classes\StdModule;
+use RobinTheHood\Stripe\Classes\Constants;
 
 class rth_stripe extends StdModule
 {
     public const VERSION = '0.1.0';
 
     public function __construct()
     {
-        $this->init('MODULE_RTH_STRIPE');
+        parent::__construct(Constants::MODULE_NAME);
```

Comment:
Unfortunately this is not possible. Since system modules and payment modules require different names.
Maybe split into: Constants::MODULE_NAME_SYSTEM and Constants::MODULE_NAME_PAYMENT